astro/src/content/docs/apis/_application-request-body.mdx

```diff
@@ -113,6 +113,14 @@ import Xmlsignaturec14nmethodValues from 'src/content/docs/_shared/_xmlSignature
 
     Required when `enabled` is set to `true`.
   </APIField>
+
+  <APIField name={props.base_field_name + '.jwtConfiguration.refreshTokenUsagePolicy'} type="String" since="1.17.0">
```

Comment:
```suggestion
  <APIField name='application.jwtConfiguration.refreshTokenUsagePolicy'} type="String" since="1.17.0" defaults="Reusable" optional>
```